src/openzaak/components/catalogi/api/filters.py

```diff
@@ -143,6 +155,7 @@ class BesluitTypeFilter(FilterSet):
             "Het INFORMATIEOBJECTTYPE van informatieobjecten waarin besluiten van dit "
             "BESLUITTYPE worden vastgelegd."
         ),
+        validators=[URLValidator()],
```

Comment:
Do I get it right that we couldn't use urls without hostname (for example '/api/v1/zaken/<uuid>') as filter parameters anymore?

Reply:
that's correct

Reply:
I don't think it is consistent with other operations since we still can include urls without hostname in the body in POST action. And it adds extra work to modify performance tests, which I don't want to, because hostname will have to be hardcoded.
But if you and @sergei-maertens are ok with it, I am also ok

Reply:
The URLs without hostname in bodies working as expected is an unintended side-effect. We always had the intention to use fully qualified URLs